src/main/java/net/mcreator/ui/component/CollapsiblePanel.java

```diff
@@ -29,122 +30,71 @@ public class CollapsiblePanel extends JPanel {
 
 	private String title = "";
 	private final TitledBorder border;
+	protected final JPanel contentHolder = new JPanel();
 
 	public CollapsiblePanel(String text, JComponent content) {
-		border = BorderFactory.createTitledBorder(title);
-		setBorder(border);
-		BorderLayout borderLayout = new BorderLayout();
-		setLayout(borderLayout);
+		this(text, content, true);
+	}
+
+	public CollapsiblePanel(String text, JComponent content, boolean allowExpandInitially) {
+		setLayout(new BorderLayout());

```

Comment:
contentHolder should also have BorderLayout to keep functionality the same as it was before

Reply:
✔️ Fixed
(After this being marked as draft again...)